tests/testthat/test-dbWriteTable.R

```diff
@@ -15,3 +15,54 @@ test_that("options work", {
   dbWriteTable(con, "dat", "dat-n.txt", sep="|", eol="\n", overwrite = TRUE)
   expect_equal(dbReadTable(con, "dat"), expected)
 })
+
+test_that("appending is error if table does not exist", {
+  if (!mysqlHasDefault()) skip("Test database not available")
+
+  con <- dbConnect(MySQL(), dbname = "test")
+  on.exit(dbDisconnect(con))
+
+  df <- data.frame(
+    str = letters[1:5],
+    num = 1:5,
+    stringsAsFactors = FALSE
+  )
+  dbWriteTable(con, "dat", df, overwrite = TRUE)
+
+  expect_error(
+    dbWriteTable(con, "dat", df, overwrite = FALSE, append = FALSE),
+    "Table dat exists"
+  )
+})
+
+test_that("temporary tables work properly", {
+  if (!mysqlHasDefault()) skip("Test database not available")
+
+  con <- dbConnect(MySQL(), dbname = "test")
+  on.exit(dbDisconnect(con))
+
+  dbGetQuery(
```

Comment:
OK, but then if there is a `temporary` argument, then my PR does not make much sense.  Btw. even then, I would not change this test, because here I test how dbWriteTable behaves with an already existing temporary table.